### DISCUSSION TITLE
Retain District capability in Meeting Guide

### DISCUSSION THREAD

**Discussion by Peter50996** [code4recovery/12-step-meeting-list#1517] [General]
Let me start by thanking you for all the hard work you do. Your work matters. It casts light on the path trekked by many of our hopeless fellows out of despair and ultimate death toward the promise of a healthy and fulfilling life.

 

I am writing because I am a bit concerned about the recent removal of the ‘Switch to Districts’ capability from the meeting app. I understand that the bulk of users of the meeting app are interested in the geographic location of a meeting and not its district. The capability to search by districts was not coded and included in the app to help persons looking for meetings. It was put there to help those who maintain the list to ensure its accuracy and for other administrative purposes. It has been my experience that the AA district is the key to maintaining the accuracy of the meeting lists. One of the most important activities of a District is to educate and assist the district’s groups and GSRs in maintaining accuracy of meeting list and registration.  Over the years, myself and other members of our district committee  have used the ‘sort by district’ capability for this purpose on a regular basis. This is probably the reason why there have been so many complaints about the removal of the capability. It would be useful to put this capability back in the meeting app.

 

You may wish to also consider two other aspects related to this concern. AA was founded on the recognition that the key to recovery is one alcoholic helping another alcoholic recover from this deadly illness. When we work with one another at the group and district level to maintain meeting lists, with the slogans and traditions as a guide, we are learning to overcome our dysfunctional ‘-ism’ and develop new, sober, behaviors.  Using technology to centralize administrative tasks may be more efficient, but it decreases their effectiveness as a tool for helping fulfill AA’s primary purpose at a local level.

 

The second aspect, related to the first, is the decision-making process related to the use of technology to advance AA primary purpose. It is easy to say ‘You should have asked before you took out that capability.’  That idea splatters against the wall of ‘How do you do that?’ I have been very impressed with NAATW’s and TiAA’s efforts to integrate and ensure both the use and teaching of the traditions in the area of technology in AA. It is a real and difficult challenge, but it would be a tragedy to circumvent. The ultimate success of technology in AA will be determined by how well it is interwoven with AA’s traditions and primary purpose. I commend you in your efforts in this regard.

**Comment by tim-rohrer** [comment]
Hi @Peter50996,

If you're referring to the mobile app Meeting Guide, and changes they've made, please contact  GSO.
If you're referring to the `12-Step-Meeting-List` plugin, the support for Districts has not been removed. The Legacy UI shows "Switch to Districts" for end users (although we really question how often this is used by people trying to find meetings). TSML UI does not show Districts, but this isn't a feature that was removed; it was just never implemented in TSML UI. Webmasters still have the full capability in the admin side of TSML to set and track district information.

**Reply by joshreisner** [reply]
that's right, and in addition, you can still search for districts and link to them using TSML UI. if you can share your URL @Peter50996 , we can show you how to form those links.

**Reply by boundary-conditions** [reply]
Hello! I was hoping to find this topic here and it's at the top of the list. I think the "Switch to Districts" setting for end users is mostly helpful for DCM's, which isn't a super common use but I would say probably a pretty important feature when it's needed. We recently switched to the TSML UI, I think it's great for most of our users but I'd like to know how to form the links to districts as well.

Thanks!

**Reply by joshreisner** [reply]
sure here's an example: https://bcyukonaa.org/meetings/?search=%22District+60%22

**Comment by Peter50996** [comment]
Tim,

Thank you very much for the information. It is very helpful.

Peter
